sdk/textanalytics/Azure.AI.TextAnalytics/src/autorest.md

```diff
@@ -47,4 +47,95 @@ directive:
   transform: >
     $.properties.id["x-nullable"] = true;
     $.properties.text["x-nullable"] = true;
+```
+
+### Add x-ms-paths section if not exists
```

Comment:
@ShivangiReja I tried my best with the transforms. If there is an easier way, please let me know